miwa.rb

```diff
@@ -0,0 +1,45 @@
+class Indicatory
```

Comment:
This class receives all inputs from the user, and should be named something like `Form` `Collector` or `Receiver`.